src/DynamoCore/Configuration/DebugModes.cs

```diff
@@ -102,33 +103,34 @@ public static void SetDebugModeEnabled(string name, bool enabled)
             DebugMode dbgMode;
             if (debugModes.TryGetValue(name, out dbgMode))
             {
-                dbgMode.Enabled = enabled;
+                dbgMode.IsEnabled = enabled;
             }
         }
+
         /// <summary>
         /// Returns a dictionary of all the debug modes
         /// </summary>
         public static Dictionary<string, DebugMode> GetDebugModes()
         {
             return debugModes;
         }
+
         /// <summary>
         /// Retrieves a debug mode
         /// </summary>
         /// <param name="name">Name of the debug mode</param>
         public static DebugMode GetDebugMode(string name)
         {
-            DebugMode dMode;
-            return debugModes.TryGetValue(name, out dMode) ? dMode : null;
+            return debugModes.TryGetValue(name, out DebugMode dMode) ? dMode : null;
```

Comment:
I think the compiler version in the Build machine does not support in-lined out variable declarations.
I had similar issues ....

Reply:
@QilongTang FYI ^

Reply:
Err from jenkins build:
`
Configuration\DebugModes.cs(124,63): error CS1003: Syntax error, ',' expected [E:\Builds\Jenkins\Dynamo_master\Dynamo\src\DynamoCore\DynamoCore.csproj]
`

Reply:
@pinzart Thanks, will revert my change here